sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -28,7 +28,6 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" />
```

Comment:
Wouldn't you need to remove the DiagnosticsSource dependency to workaround this issue? How does removing the unsafe help in this case?

Reply:
Azure Functions runtime forcibly and silently downgrades dependencies independently from what .deps.json has. I didn't downgrade the dependency so that customers outside functions still get full distributed tracing experience.

Reply:
So with this change our libraries function correctly with the downgraded DignosticsSource dependency? I suspect that will end up biting us if we depend on that behavior but perhaps it is a OK for a short-term fix.

Reply:
No, the functionality degrades "gracefully" but it's a temporary targeted tactical fix. I'm continuing to pursue the right fix in the meantime.